.swiftlint.yml

```diff
@@ -49,9 +52,31 @@ type_name:
    warning: 80
    error: 100
 
+file_header:
+  required_pattern: |
+                    \/\/
+                    \/\/  .*?\.swift
+                    \/\/  .*?
+                    \/\/
+                    \/\/  Copyright © \d{4} DuckDuckGo\. All rights reserved\.
+                    \/\/
+                    \/\/  Licensed under the Apache License, Version 2\.0 \(the \"License\"\);
+                    \/\/  you may not use this file except in compliance with the License\.
+                    \/\/  You may obtain a copy of the License at
+                    \/\/
+                    \/\/  http:\/\/www\.apache\.org\/licenses\/LICENSE-2\.0
+                    \/\/
+                    \/\/  Unless required by applicable law or agreed to in writing, software
+                    \/\/  distributed under the License is distributed on an \"AS IS\" BASIS,
+                    \/\/  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.
+                    \/\/  See the License for the specific language governing permissions and
+                    \/\/  limitations under the License\.
+                    \/\/
+
 # General Config
 excluded:
   - Carthage
   - DuckDuckGo/UserText.swift
+  - fastlane/SnapshotHelper.swift
```

Comment:
Ignoring this as it's a file we didn't write, we want to preserve its original header value.